namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-apply-for-legalaid-staging/resources/route53.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_route53_zone" "apply_for_legal_aid_route53_zone" {
+  name = "staging.apply-for-legal-aid.service.justice.gov.uk"
```

Comment:
Can you please create a zone some thing like `apply-for-legal-aid.service.justice.gov.uk`
Can see more guidance about creating a Zone [here][https://user-guide.cloud-platform.service.justice.gov.uk/tasks.html#defining-the-dns-zone]